### PR TITLE
added limitation

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-db1f18dfb1/how-to/).

**Comment by cyrichardson** [comment]
Wil do!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by cyrichardson** [comment]
Suggestions in the following file, @stephamon:
[rackspace-email-archiving-known-limitations_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1653376/rackspace-email-archiving-known-limitations_cyrEdits.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-08984355f0/how-to/)

File: content/rackspace-email-archiving/rackspace-email-archiving-known-limitations.md [hunk 1]
```diff
@@ -1,55 +1,60 @@
...
+#### Rules
 
 If archiving is enabled for the domain but an address at that domain has
 a rule or filter to delete messages from specific senders, the deleted
 messages are *not* archived.
+
+
+#### Spam
+
+If a message is flagged as spam, the message is not be archived.
```
Comment (catlook) [inline content/rackspace-email-archiving/rackspace-email-archiving-known-limitations.md]: ... is not archived. ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cbe1683ec9/how-to/).

**Pull request merged**